core/storage/feedback/gae_models.py

```diff
@@ -113,14 +117,26 @@ def get_by_exp_and_thread_id(cls, exploration_id, thread_id):
 
     @classmethod
     def get_threads(cls, exploration_id):
-        """Returns an array of threads associated to the exploration.
+        """Returns a list of threads associated to the exploration, ordered by
+        their last updated field.
 
         Does not include the deleted entries.
         """
         return cls.get_all().filter(
             cls.exploration_id == exploration_id).order(
                 cls.last_updated).fetch(feconf.DEFAULT_QUERY_LIMIT)
 
+    @classmethod
+    def get_latest_updated_threads(cls, exploration_id):
```

Comment:
This method and the previous one look almost identical. Why not pass in a 'limit' arg to the previous one which defaults to feconf.DEFAULT_QUERY_LIMIT?